systemverilog/SystemVerilog.g4

```diff
@@ -0,0 +1,3757 @@
+// Author: Mustafa Said Ağca
+// License: MIT
+
+grammar SystemVerilog;
+import SystemVerilogLexerRules;
+
+/*******************
+	PARSER RULES
+*******************/
+
+// A.1.1 Library source text
+
+/*library_text
+	: (library_description)*
+	;
+
+library_description
+	: library_declaration
+	| include_statement
+	| config_declaration
+	| ';'
+	;
+
+library_declaration
+	: 'library' library_identifier FILE_PATH_SPEC (',' FILE_PATH_SPEC)* ('-incdir' FILE_PATH_SPEC (',' FILE_PATH_SPEC)*)? ';'
+	;
+
+include_statement
+	: 'include' FILE_PATH_SPEC ';'
+	;
+*/
+// A.1.2 SystemVerilog source text
+
+source_text
+	: (timeunits_declaration)? (description)* EOF
```

Comment:
No need for parentheses for a single expression:

```suggestion
	: timeunits_declaration? description* EOF
```

Here and in entire grammar.